feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/followrequest/FollowerRequestFragment.java

```diff
@@ -0,0 +1,62 @@
+package com.example.feelslikemonday.ui.followrequest;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProviders;
+
+import com.example.feelslikemonday.R;
+import com.example.feelslikemonday.ui.followrequest.ArraryAdapter.RequestList;
+import com.example.feelslikemonday.model.User;
+
+import java.util.ArrayList;
+
+public class FollowerRequestFragment extends Fragment {
+    private ListView userList;
+    private ArrayAdapter<User> userAdapter;
+    private ArrayList<User> userDataList;
+    private RequestList requestList;
+
+    private FollowerRequestViewModel followerViewModel;
+
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             ViewGroup container, Bundle savedInstanceState) {
+
+        followerViewModel = ViewModelProviders.of(this).get(FollowerRequestViewModel.class);
+
+        View root = inflater.inflate(R.layout.fragment_follower_request, container, false);
+
+        userList = root.findViewById(R.id.request_username_list);
+        userDataList = new ArrayList<>();
+        //test
+        String username = "your lovely follower";
```

Comment:
In the diagrams, the username comes out as "your lovely". Would it be possible to make the text smaller so that the whole username fits? I can also add limits to the username in the when creating a user

Reply:
I try to make the textview scrollable.